### ISSUE TITLE
 F tensorflow/stream_executor/cuda/cuda_dnn.cc:503] Check failed: cudnnSetTensorNdDescriptor(handle_.get(), elem_type, nd, dims.data(), strides.data()) == CUDNN_STATUS_SUCCESS (3 vs. 0)batch_descriptor: {count: 0 feature_map_count: 512 spatial: 2 30 39  value_min: 0.000000 value_max: 0.000000 layout: BatchDepthYX} Aborted (core dumped)

### ISSUE THREAD

**Issue by ArpithaFAU** [keras-team/keras#13984]
Please make sure that the boxes below are checked before you submit your issue. If your issue is an implementation question, please ask your question on [StackOverflow](http://stackoverflow.com/questions/tagged/keras) or [join the Keras Slack channel](https://keras-slack-autojoin.herokuapp.com/) and ask there instead of filing a GitHub issue.

Thank you!

- [x] Check that you are up-to-date with the master branch of Keras. You can update with:
pip install git+git://github.com/keras-team/keras.git --upgrade --no-deps

- [x] If running on TensorFlow, check that you are up-to-date with the latest version. The installation instructions can be found [here](https://www.tensorflow.org/get_started/os_setup).

- [ ] If running on Theano, check that you are up-to-date with the master branch of Theano. You can update with:
pip install git+git://github.com/Theano/Theano.git --upgrade --no-deps

- [x] Provide a link to a GitHub Gist of a Python script that can reproduce your issue (or just copy the script here if it is short).

**Comment by ArpithaFAU** [comment]
Can anyone help me with this error?

initialiser = 'glorot_uniform'
reg = regularizers.l2(0.01)
input_model = Input(shape=( frames,None,None, 1), name='input_model')

first_3dconv = Conv3D(32, kernel_size=(3,3,3),strides=2,padding='same', kernel_initializer = initialiser,kernel_regularizer = reg )(input_model)
first_dp = Dropout(0.5)(first_3dconv)
first_bn = BatchNormalization()(first_dp)
first_act = Activation('relu')(first_bn)
#first_pool = MaxPooling3D(pool_size=(1,2,2), strides=(1, 2, 2) ,padding='same')(first_3dconv)
sec_3dconv = Conv3D(64,kernel_size=(3,3,3),strides = 2, padding='same', kernel_initializer = initialiser, kernel_regularizer = reg)(first_act)
#sec_pool = MaxPooling3D(pool_size=(1,2,2), strides=(1,2,2), padding='same')(sec_3dconv)
sec_dp = Dropout(0.5)(sec_3dconv)
sec_bn = BatchNormalization()(sec_dp)
sec_act = Activation('relu')(sec_bn)
third_3dconv = Conv3D(128,kernel_size=(3,3,3),strides = 2, padding='same', kernel_initializer = initialiser, kernel_regularizer = reg)(sec_act)
third_dp = Dropout(0.5)(third_3dconv)
third_bn = BatchNormalization()(third_dp)
third_act = Activation('relu')(third_bn)
fourth_3dconv = Conv3D(256,kernel_size=(3,3,3), strides = 2,padding='same', kernel_initializer = initialiser, kernel_regularizer = reg)(third_act)
fourth_dp = Dropout(0.5)(fourth_3dconv)
fourth_bn = BatchNormalization()(fourth_dp)
fourth_act = Activation('relu')(fourth_bn)
fifth_3dconv = Conv3D(512,kernel_size=(3,3,3), strides = 2,padding='same', kernel_initializer = initialiser, kernel_regularizer = reg)(fourth_act)
fifth_dp = Dropout(0.5)(fifth_3dconv)
fifth_bn = BatchNormalization()(fifth_dp)
fifth_act = Activation('relu')(fifth_bn)
ninth_3dconv = Conv3D(64,kernel_size=(1,1,1),strides= 1, padding='same')(fifth_act)
d1= Dropout(0.5)(ninth_3dconv)
b1 = BatchNormalization() (d1)
a1 = Activation('relu')(b1)
tenth_3dconv = Conv3D(num_classes,kernel_size=(1,1,1), strides=1,padding='same')(a1)
d2= Dropout(0.5)(tenth_3dconv)
b2 = BatchNormalization()(d2)
gpool = GlobalMaxPooling3D()(b2)
out = Activation('sigmoid')(gpool)

This is the model that i am using.

I am trying to train this FCN model with variable input size and i am using only a batch size of 1 as my data is sequences.

The dimension is frames x height x width x channel. The frames used are constant

**Comment by blackvvine** [comment]
I'm experiencing the same error with Tensorflow 2.2 and CUDA 10.1

```python
def model_create_lstm(submodel=model_cnn):
    N_SUPERFLOWS = None
        
    def add_distributed(out, l):
        out = KL.TimeDistributed(l, name="td_" + l.name)(out)
        return out

    h_input = KL.Input(shape=(N_SUPERFLOWS, N_HEADERS, PACKET_CUTOFF), name="H_In")
    h_out = h_input
    
    h_out = add_distributed(h_out, submodel)
    print(h_out.shape)
    
    f_input = KL.Input(shape=(N_SUPERFLOWS,), name="F_In")
    f_out = f_input
    
    
    f_out = KL.Lambda(KB.expand_dims, output_shape=(N_SUPERFLOWS,1), name="F_Reshape")(f_out)
    
    #f_out = add_distributed(f_out, KL.Reshape(target_shape=(N_SUPERFLOWS,1), name="F_Reshape"))
    
    #f_out = KL.Masking(mask_value=MASK_VALUE, name="F_Mask")(f_input)
    
    out = KL.concatenate([h_out, f_out])
    
    out = add_distributed(out, KL.Dense(64, activation=ACTIVATION, name="Dense1"))
    out = add_distributed(out, KL.Dense(64, activation=ACTIVATION, name="Dense2"))
    
    out = CuDNNLSTM(50, name="LSTM")(out)
    out = Dense(len(ensemble), activation='softmax')(out)
    
    return Model(inputs=[h_input, f_input], outputs=out, name="model-cnn-lstm")

```

Here's the error:

```
2020-08-08 11:35:54.186630: W tensorflow/core/kernels/gpu_utils.cc:49] Failed to allocate memory for convolution redzone checking; skipping this check. This is benign and onl
y means that we won't check cudnn for out-of-bounds reads and writes. This message will only be printed once.                                                                
2020-08-08 11:35:54.186739: F tensorflow/stream_executor/cuda/cuda_dnn.cc:534] Check failed: cudnnSetTensorNdDescriptor(handle_.get(), elem_type, nd, dims.data(), strides.dat
a()) == CUDNN_STATUS_SUCCESS (9 vs. 0)batch_descriptor: {count: 9528 feature_map_count: 256 spatial: 1 1535  value_min: 0.000000 value_max: 0.000000 layout: BatchDepthYX}
```

Any help would be appreciated.

**Comment by bhkann** [comment]
Same error for me too - any help would be much appreciated!

**Comment by tilakrayal** [comment]
Could you please update to latest stable tf and keras versionand let us know if you are facing the same error. Thanks!

**Comment by jvishnuvardhan** [comment]
@ArpithaFAU Is this still an issue for you. Sorry for late response.

Can you please share a simple standalone code to reproduce the issue? Also, check your code with the recent TF versions (`2.6` and `tf-nightly`) and let us know whether the issue persists with recent TF versions. Thanks!

**Comment by ArpithaFAU** [comment]
No my issue is fixed. I'm currently using tf 2.4 with cuda 11.0
